csp/gitprojectsettings.csp

```diff
@@ -634,7 +654,12 @@ $(function()
     $(document).on('click', "#goToWebUI", function(e){
         e.preventDefault();
         console.log($("#webuiURL")[0].innerText);
```

Comment:
I know you didn't add this, but we should make sure there aren't calls to console.log in here (in general) because those will probably break in Studio.